tests/conftest.py

```diff
@@ -62,11 +69,21 @@ def get_data_from_id(self, contact_id):
         except:
             logger.exception('foo')
         return matches
+
+    def get_data_to_match_hash(self, match_term):
+        idx = rtree.index.Index('/Users/dan/tmp/rtree')
+        return list(filter(lambda obj: obj['updatetoken'] == match_term,
```

Comment:
Could you add a comment on this one - I'm not familiar with lambda functions, though they look similar to a JS syntax I know, but I think a comment would be useful as to what this is doing as I suspect others won't know lambda.

Reply:
I made the code clearer.